aimage/views.py

```diff
@@ -51,6 +55,49 @@ async def _check_if_can_delete(self, interaction: discord.Interaction):
 
         guild = interaction.guild
         member = guild.get_member(interaction.user.id)
-        can_delete = await self.bot.is_owner(member) or interaction.channel.permissions_for(member).manage_messages
+        is_staff = await self.bot.is_mod(member) or await self.bot.is_admin(member) or await self.bot.is_owner(member)
+
+        return is_og_user or is_staff
+
+    async def get_params_embed(self) -> discord.Embed:
+        output_dict = OrderedDict()
+        prompts, params = self.info_string.rsplit("Steps: ", 1)
```

Comment:
when using ai horde fallback, it breaks here

eg. 
```
1girl
AI Horde image. Seed: 1798293003, Model: Anything Diffusion
```

Reply:
Oh right, I'll edit it later